src/helpers.coffee

```diff
@@ -3,6 +3,21 @@
 # arrays, count characters, that sort of thing.
 
 
+# pollyfill for async/await
```

Comment:
I don’t want to add any polyfills. Is there a reason why this is required? Did the method I suggested in my comments not work for some reason?

Let’s try to find a way to run the tests in Node 6.9.1 _and_ Node 7+ harmony, with them working in both cases, without polyfills. I think we already found a way, unless you’re telling me there’s something wrong with my approach.